src/metabase/automagic_dashboards/populate.clj

```diff
@@ -43,9 +44,23 @@
         :location "/")
       (create-collection! "Automatically Generated Dashboards" "#509EE3" nil nil)))
 
-(def colors
+(defn- whitelabeled-colors
+  "Get color from application-colors or default color."
+  [default-colors-map]
+  (vec (for [[whitelabeled-color-name default-color] default-colors-map]
+         (get (public-settings/application-colors) whitelabeled-color-name default-color))))
+
+(defn colors
   "Colors used for coloring charts and collections."
-  ["#509EE3" "#9CC177" "#A989C5" "#EF8C8C" "#f9d45c" "#F1B556" "#A6E7F3" "#7172AD"])
+  []
+  (whitelabeled-colors {:brand "#509EE3"
+                        :accent1 "#88BF4D"
+                        :accent2 "#A989C5"
+                        :accent3 "#EF8C8C"
+                        :accent4 "#F9D45C"
+                        :accent5 "#F2A86F"
+                        :accent6 "#98D9D9"
+                        :accent7 "#7172AD"}))
```

Comment:
Change some colors to match `colors.ts`, they were close but weren't the same color.
https://github.com/metabase/metabase/blob/8b341b1ac3b1c95d891058966bfb30b264b9285f/frontend/src/metabase/lib/colors.ts#L10-L18

From eye-balling I get that the colors were defined like this
`[brand accent1 accent2 accent3 accent4 accent5 accent6 accent7]`